src/main/java/com/dazednconfused/catalauncher/backup/SaveManager.java

```diff
@@ -129,16 +151,11 @@ public static boolean saveFilesExist() {
      * */
     public static Optional<File> getLatestSave() {
 
-        if (!saveFilesExist()) {
+        if (!saveFilesExist() || (getLastModifiedValidSave().orElse(null) == null)) {
```

Comment:
Since `getLastModifiedValidSave` returns an `Optional`, we could be checking for `Optional#empty` outright instead of unwrapping and checking for `null`, no?